roles/development/files/bin/make-changelog.py

```diff
@@ -81,11 +98,41 @@ def get_pull_info(username, project, number):
     body = response.json()
     title = body['title']
     author = body['user']['login']
-    link = pull_url_for(username, project, number)
+    link = pull_url_for_github(username, project, number)
     return title, author, link
 
 
-def main(username, project, version):
+def get_pull_info_pagure(username, project, number):
+    template = 'https://pagure.io/api/0/{project}/pull-request/{number}'
+    url = template.format(project=project, number=number)
+    response = github_session.get(url)
+    body = response.json()
+    title = body['title']
+    author = body['user']['name']
+    link = pull_url_for_pagure(project, number)
+    return title, author, link
+
+
+def get_pull_request_pagure(project, commit):
```

Comment:
if this returns more than one pull request, it needs to be called ``get_pull_requests_pagure``.  Plural to indicate that it returns a collection.